static/js/narrow.js

```diff
@@ -993,7 +993,9 @@ function handle_post_narrow_deactivate_processes() {
     if (page_params.search_pills_enabled) {
         search_pill_widget.widget.clear(true);
     }
-
+    if (message_lists.current.empty()) {
+        narrow_banner.show_empty_narrow_message();
+    }
```

Comment:
This probably deserves a comment noting that `reset_ui_state` will have cleared this.